test/tests/functional/pbs_acl_groups.py

```diff
@@ -56,34 +56,7 @@ def test_acl_grp_queue(self):
         j = Job(TEST_USER1, attrs=a)
         try:
             jid = self.server.submit(j)
-        except PbsSubmitError, e:
-            self.assertFalse('qsub: Unauthorized Request' in e.msg[0])
-
-    def test_acl_grp_resv(self):
-        """
-        Set acl_groups on a reservation queue and submit a job
-        from a user for whom the set group is a secondary group
-        """
-        r = Reservation(TEST_USER)
-        rstart = int(time.time()) + 10
-        rend = int(time.time()) + 360
-        a = {'reserve_start': rstart,
-             'reserve_end': rend}
-        r.set_attributes(a)
-        rid = self.server.submit(r)
-        self.server.expect(
-            RESV, {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}, id=rid)
-        resv_queue = rid.split('.')
-        rid = resv_queue[0]
-        a = {'acl_group_enable': 't', 'acl_groups': TSTGRP1}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, id=rid)
-        self.server.manager(MGR_CMD_UNSET, QUEUE, 'acl_user_enable', id=rid)
-        self.server.manager(MGR_CMD_UNSET, QUEUE, 'acl_users', id=rid)
-        a = {'queue': rid}
-        j = Job(TEST_USER1, attrs=a)
-        try:
-            jid = self.server.submit(j)
-        except PbsSubmitError, e:
+        except PbsSubmitError as e:
             self.assertFalse('qsub: Unauthorized Request' in e.msg[0])
```

Comment:
This will fail in API mode because the message 'qsub:' is added by qsub.  Just look for 'Unauthorized Request'